### ISSUE TITLE
Automatically copy assets to bin folder

### ISSUE THREAD

**Issue by kevinfoley** [fdorg/flashdevelop#1587]
The other major ActionScript IDEs all support selecting asset folders from the project root that will be copied into the `bin` directory at compile time. This means that projects started in other IDEs will almost inevitably have a folder called `assets` in the project root that is supposed to get copied into `bin`, and usually the `bin` folder will be excluded from the repository. As far as I know, FD doesn't have a way to copy files into `bin` at compile time. This forces any team member working with FlashDevelop to copy the files manually, making it more difficult to keep the project synchronized.

FD should support copying `assets` or other resource directories into `bin` at compile time so that FD users can work with existing ActionScript repositories more easily.

**Comment by Neverbirth** [comment]
Yeah, I agree on this one... I started the work on this back in the day, but I stopped working on mobile projects completely so far, so my priorities shifted completely.
